lib/ansible/modules/cloud/google/gce_instance_template.py

```diff
@@ -347,7 +347,8 @@ def create_instance_template(module, gce):
     bad_perms = []
     if service_account_permissions:
         for perm in service_account_permissions:
-            if perm not in gce.SA_SCOPES_MAP:
+#            if perm not in gce.SA_SCOPES_MAP:
```

Comment:
Please, don't leave the previous version in comments. We already have Git for showing us history.

Reply:
@webknjaz thanks for your comment.
I updated.
This is the first time for me.

Reply:
Thanks!